src/components/Particles/Particles.tsx

```diff
@@ -139,5 +139,11 @@ export function ParticlesComp(): JSX.Element {
         loadSlim(engine);
         loadBubblesPreset(engine);
     }, []);
-    return <Particles className={styles.particles} init={particlesInit} options={options} />;
+    return (
+        <Particles
+            className={styles.particles}
+            init={particlesInit}
+            options={options as RecursivePartial<typeof options>}
+        />
+    );
```

Comment:
I don't think we should sneak this into a PR for a netlify backend.

Reply:
Good point, I will put this in own PR, thanks!